store/instagram-images.ts

```diff
@@ -15,14 +34,14 @@ export const instagramStore = {
     }
   },
   actions: {
-    async updateInstagramImages ({ commit, rootState }, data) {
-      let mainImageResource = rootState.storeView && rootState.storeView.storeCode ? `banners/${rootState.storeView.storeCode}_instagram-images` : `instagram-images`
```

Comment:
can you please create a bug and add a screenshot of this change. i want to make sure we are not removing the built in caching of images.